test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetVerifyCommandTest.cs

```diff
@@ -87,7 +87,7 @@ public void VerifyCommand_WithAuthorSignedPackage_FailsGracefully()
                 {
                     Assert.True(_successCode == result.ExitCode, result.AllOutput);
                     Assert.True(result.Success);
-                    Assert.Contains("Successfully verified package(s).", result.AllOutput);
+                    Assert.Contains("Successfully verified package", result.AllOutput);
```

Comment:
Can you update this expected result to [include the package ID](https://github.com/NuGet/NuGet.Client/pull/2138/files#diff-e397ea436b2c2398ac6072cb275fc524R666)?